docs/src/assets/chainrules.scss

```diff
@@ -63,26 +63,23 @@ html:not(.theme--documenter-dark) body #documenter {
       }
     }
   }
-  
+
   // Text
 
   article#documenter-page a.docs-heading-anchor {
     color: #194E82;
   }
 
   // Code
-  
+
   code {
     color: inherit;
-    
+
     .hljs-meta { color: #4595D1; }
-  
+
     .hljs-keyword { color: #194E82; }
   }
 
-  pre, code {
-    font-family: "Liberation Mono", "Consolas", "DejaVu Sans Mono", "Ubuntu Mono", "andale mono", "lucida console", monospace;
-  }
```

Comment:
Pretty sure we can just keep this and replace `font-family` with just "JuliaMono".